generation/tasks/RivetMerge.py

```diff
@@ -74,31 +81,39 @@ def remote_path(self, *path):
 
     def output(self):
         return self.remote_target(
-            "{INPUT_FILE_NAME}.yoda".format(INPUT_FILE_NAME=str(self.campaign))
+            "_".join(sorted(self.rivet_analyses))+".yoda",
         )
 
-    def mergeSingleYodaChunk(self, inputfile_list, inputfile_chunk=None):
-
+    def mergeSingleYodaChunk(self, inputfile_list, inputfile_chunk=None, workdir=None, equiv=False):
         print("-------------------------------------------------------")
         print("Starting merging of chunk {}".format(inputfile_chunk))
         print("-------------------------------------------------------")
 
-        # data
-        _my_input_file_name = str(self.campaign)
+        # Check for excessive weights in the YODA files
+        skip_files = set()
+        for _yoda_file in inputfile_list:
+            histos = yoda.read(_yoda_file, asdict=True, patterns=["/RAW/_EVTCOUNT"])
+            evt_count = histos.get("/RAW/_EVTCOUNT", None)
+            if evt_count is None:
+                logger.warning(f"No event count histogram found in {_yoda_file}! Using it anyways...")
+                continue
+            if evt_count.relErr() > self.skip_max_uncertainty:
+                logger.warning(
```

Comment:
Shouldn't this rather be an info or debug message?

Reply:
I can make it an info message. I think users should be made aware when we're skipping files.